CRM/Utils/Number.php

```diff
@@ -88,30 +88,32 @@ public static function createTruncatedDecimal($keyValue, $precision) {
   public static function formatUnitSize($size, $checkForPostMax = FALSE) {
     if ($size) {
       $last = strtolower($size{strlen($size) - 1});
+      $_size = (int)$size;
```

Comment:
@Jazz-Man i think you can do `$size = (float) $size;`

Reply:
ok, I'll try to fix it